configs/efficientdet/retinanet_effd0_bifpn_1x_coco.py

```diff
@@ -0,0 +1,162 @@
+    allowed_border=-1,
+    pos_weight=-1,
+    debug=False)
+test_cfg = dict(
+    nms_pre=1000,
+    min_bbox_size=0,
+    score_thr=0.05,
+    nms=dict(type='nms', iou_thr=0.5),
+    max_per_img=100)
+# model training and testing settings
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = './data/coco/'
+img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.374], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(
+        type='Resize',
+        img_scale=(input_size, input_size),
+        ratio_range=(0.1, 2.0),
+        keep_ratio=True),
+    dict(type='RandomCrop', crop_size=(input_size, input_size)),
+    dict(type='RandomFlip', flip_ratio=0.5),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='Pad', size=(input_size, input_size)),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels'])
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='MultiScaleFlipAug',
+        img_scale=(input_size, input_size),
+        flip=False,
+        transforms=[
+            dict(type='Resize', keep_ratio=False),
+            dict(type='Normalize', **img_norm_cfg),
+            dict(type='RandomFlip'),
+            dict(type='ImageToTensor', keys=['img']),
+            dict(type='Collect', keys=['img']),
+        ])
+]
+data = dict(
+    samples_per_gpu=32,
+    workers_per_gpu=6,
+    train=dict(
+        type='RepeatDataset',
+        times=1,
+        dataset=dict(
+            type=dataset_type,
+            ann_file=data_root + 'annotations/instances_train2017.json',
+            img_prefix=data_root + 'train2017',
+            pipeline=train_pipeline
+        )
+    ),
+    val=dict(
+        type=dataset_type,
+        ann_file=data_root + 'annotations/instances_val2017.json',
+        img_prefix=data_root + 'val2017',
+        test_mode=True,
+        pipeline=test_pipeline),
+    test=dict(
+        type=dataset_type,
+        ann_file=data_root + 'annotations/instances_val2017.json',
+        img_prefix=data_root + 'val2017',
+        test_mode=True,
+        pipeline=test_pipeline))
+# optimizer
+optimizer = dict(type='SGD', lr=0.08, momentum=0.9, weight_decay=0.00004)
+optimizer_config = dict(grad_clip=dict(max_norm=10, norm_type=2))
+# learning policy
+lr_config = dict(
+    policy='CosineRestart',
+    periods=[num_epochs],
+    min_lr=0.00001,
+    min_lr_ratio=None,
+    warmup='linear',
+    warmup_iters=5,
+    warmup_ratio=0.0001,
+    warmup_by_epoch=True
+)
+checkpoint_config = dict(interval=1)
+# yapf:disable
+log_config = dict(
+    interval=500,
+    hooks=[
+        dict(type='TextLoggerHook'),
+    ])
+# yapf:enable
+# runtime settings
+total_epochs = num_epochs
+dist_params = dict(backend='nccl')
+log_level = 'INFO'
+work_dir = 'outputs'
+load_from = None
+resume_from = None
+workflow = [('train', 1)]
```

Comment:
Is it possible to derive this config file from existing ones and override some parameters instead of defining everything here? Or difference is too large?

Reply:
After attempting to use existing models from the _base_ folder, it looks like the difference is too large to do.